build/patches/Experimental-user-scripts-support.patch

```diff
@@ -10405,15 +10405,15 @@ new file mode 100755
 +  enabled_ = params.allow_userscript;
 +  if (!enabled_) return;
 +
-+  if (loaded_ == false) {
-+    loaded_ = true;
```

Comment:
Is `loaded_` going to be inconsistent without this? Do we use `loaded_` anywhere else or can it be removed?

Reply:
> or can it be removed?

I have already removed it, look at the line 10486